docs/guides/web/apache_hardened_webserver/index.md

```diff
@@ -9,143 +9,98 @@ tags:
+These examples use a private IP address to simulate a public one, but completing all of this _could_ be done with a one-to-one NAT on the hardware firewall and connecting the web server to that hardware firewall, rather than to the gateway router, with a private IP address.
 
-Explaining that requires digging into the hardware firewall shown below, and since that is outside of the scope of this document, it is better to stick with our example of a simulated public IP address.
+Explaining that requires digging into the hardware firewall shown, and that is outside of the scope of this document.
 
 ## Conventions
 
-* **IP Addresses:** We are simulating the public IP address here with a private block: 192.168.1.0/24 and we are using the LAN IP address block as 10.0.0.0/24  In other words, it cannot be routed over the Internet. In reality, neither IP block can be routed over the Internet as they are both reserved for private use, but there is no good way to simulate the public IP block, without using a real IP address that is assigned to some company. Just remember that for our purposes, the 192.168.1.0/24 block is the "public" IP block and the 10.0.0.0/24 is the "private" IP block.
-* **Hardware Firewall:** This is the firewall that controls access to your server room devices from your trusted network. This is not the same as our _iptables_ firewall, though it could be another instance of _iptables_ running on another machine. This device will allow ICMP (ping) and SSH (secure shell) to our trusted devices. Defining this device is outside of the scope of this document. The author has used both [PfSense](https://www.pfsense.org/) and [OPNSense](https://opnsense.org/) and installed on dedicated hardware for this device with great success. This device will have two IP addresses assigned to it. One that will connect to the Internet router's simulated public IP (192.168.1.2) and one that will connect to our local area network, 10.0.0.1.
-* **Internet Router IP:** We are simulating this with 192.168.1.1/24
-* **Web Server IP:** This is the "public" IP address assigned to our web server. Again, we are simulating this with the private IP address 192.168.1.10/24
-
-![Hardened Webserver](images/hardened_webserver_figure1.jpeg)
-
-The diagram above shows our general layout. The _iptables_ packet-based firewall runs on the web server (shown above).
+* **IP Addresses:** simulating the public IP address here with a private block: 192.168.1.0/24 and, using the LAN IP address block 10.0.0.0/24. Routing these IP blocks over the Internet is not possible because they are for private use, but simulating public IP block is not possible without the use of a real IP address assigned to some company or organization. Just remember that for our purposes, the 192.168.1.0/24 block is the "public" IP block and the 10.0.0.0/24 is the "private" IP block.
```

Comment:
```suggestion
* **IP Addresses:** simulating the public IP address here with a private block: 192.168.1.0/24, and using the LAN IP address block 10.0.0.0/24. Routing these IP blocks over the Internet is not possible because they are for private use, but simulating public IP block is not possible without the use of a real IP address assigned to some company or organization. Just remember that for our purposes, the 192.168.1.0/24 block is the "public" IP block and the 10.0.0.0/24 is the "private" IP block.
```